cmd/nomos/status/client.go

```diff
@@ -279,7 +279,11 @@ func (c *ClusterClient) multiRepoClusterStatus(ctx context.Context, cs *ClusterS
 		}
 	}
 
-	if len(repoSyncs) != 0 {
+	// We need to do an empty slice check on namespaceRGs as well in case there was an error
+	// only getting the resourceGroups. If there were no errors getting both, they are guranteed
+	// to be identical in length due to how we create the namespaceRGs slice by initializing it to
+	// be the same length as repoSyncNsAndNames before populating it.
+	if len(repoSyncs) != 0 && len(namespaceRGs) != 0 {
 		var repos []*RepoState
```

Comment:
This is implementation detail in `namespaceRGs` creation and not guaranteed to be always true (someone might change the function and break it in the future). Check that the length are identical and append an error if not, so that we won't hit panic errors.

Reply:
Good idea. Added a check to also ensure both slices are of the same length.